server/src/metadata/migrations/1698308729020-removeMetadataSoftDelete.ts

```diff
@@ -0,0 +1,61 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class RemoveMetadataSoftDelete1698308729020
+  implements MigrationInterface
+{
+  name = 'RemoveMetadataSoftDelete1698308729020';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "metadata"."field_metadata" DROP COLUMN "deleted_at"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "metadata"."object_metadata" DROP COLUMN "deleted_at"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "metadata"."data_source_metadata" ALTER COLUMN "id" SET DEFAULT uuid_generate_v4()`,
```

Comment:
It seems it's adding this default value again in this migration, for some reason it was removed and created a bug last week